### PR TITLE
Fix a few flake8 errors that only occur on EL 7.

### LINKED ISSUE #1269: flake8 errors happen during EPEL 7 build
The version of flake8 used in EPEL 7 is upset about a few things:

```
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/buildsys.py:252:17: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/services/releases.py:64:13: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/services/releases.py:68:13: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/views/generic.py:224:101: E501 line too long (116 > 100 characters)
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/server/consumers/masher.py:558:33: E126 continuation line over-indented for hanging indent
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/tests/server/consumers/test_masher.py:104:101: E501 line too long (112 > 100 characters)
/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/tests/server/consumers/test_masher.py:399:18: E121 continuation line indentation is not a multiple of four
```

We could either find a way to appease both versions of flake8, or we could adjust the style test to detect which version of flake8 is present and only test with the newer one that we like. Doesn't matter to me.

### PR DESCRIPTION
**Pull request by bowlofeggs**
fixes #1269

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: bodhi/server/views/generic.py [hunk 1]
```diff
@@ -221,7 +221,9 @@ def api_version(request):
 def notfound_view(context, request):
     """ Automatically redirects to slash-appended routes.
 
-    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#redirecting-to-slash-appended-rou
+    Note: The URL below spans two lines.
+    http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html
+    #redirecting-to-slash-appended-rou
```
Comment (pypingou) [inline bodhi/server/views/generic.py]: Anyway to make flake8 ignore this one?

**Comment by pypingou** [comment]
Except for the URLs which I think we should keep in one line, 👍 for me

That is imho though, so feel free to disagree :)

File: bodhi/server/views/generic.py (hunk 1 above)
Comment (jeremycline) [reply bodhi/server/views/generic.py]: It's worth noting that that fragment doesn't even exist. You _could_ just drop it.

Comment (bowlofeggs) [reply bodhi/server/views/generic.py]: Good point, I'll just drop the missing fragment since it doesn't exist.

**Pull request merged**